velox/expression/tests/FuzzerRunner.h

```diff
@@ -106,16 +106,139 @@ class FuzzerRunner {
     return output;
   }
 
-  static const std::unordered_map<
+  static std::unordered_map<
       std::string,
       std::vector<facebook::velox::exec::FunctionSignaturePtr>>
-      kSpecialForms;
+      specialFormSignatures;
```

Comment:
Any particular reason to change this name. The original names was nice and matched naming convention.

Reply:
Because I removed "const" of this variable, and the function appendSpecialForms() already has a parameter called specialForms, so I changed the name of this variable to specialFormSignatures.

Reply:
I think we can keep this as const and preserve the original name by adding getSignaturesForCast helper function and using it like so:

https://github.com/facebookincubator/velox/compare/main...mbasmanova:velox-1:export-D43686197?expand=1

```
        {
            "cast",
            getSignaturesForCast(),
        },
```